timeline-chart/src/layer/time-graph-axis.ts

```diff
@@ -17,17 +17,21 @@ export class TimeGraphAxis extends TimeGraphLayer {
     private _keyUpHandler: { (event: KeyboardEvent): void; (this: Document, ev: KeyboardEvent): any; };
     private _keyDownHandler: { (event: KeyboardEvent): void; (this: Document, ev: KeyboardEvent): any; };
 
-    constructor(id: string, protected style?: { color?: number, lineColor?: number }) {
+    constructor(id: string, protected style?: { color?: number, lineColor?: number, isTop?: boolean }) {
```

Comment:
Sorry, we had further discussion and we would suggest that we use the vertical-align string property from CSS (verticalAlign in code):

https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align

So if you align to the top, that means you stick to the top of your layout cell, then the ticks should be on top.
If you align to the bottom the ticks should be at the bottom.

Looking at the link above, it seems their default is 'baseline' and in the example that aligns to the top. So perhaps we could default to 'top'.

<pre>
+---|---|---|---+
|   1   2   3   |  verticalAlign = 'top'
|               |
+---------------+

+---------------+
|               |
|   1   2   3   |  verticalAlign = 'bottom'
+---|---|---|---+
</pre>